python/tests/res/fm/test_shell.py

```diff
@@ -0,0 +1,322 @@
+import unittest
+import os
+import os.path
+
+from ecl.util.test import TestAreaContext
+from tests import ResTest
+from res.fm.shell import Shell
+
+
```

Comment:
In this test please also import the small functions from `shell/__init__.py`:
```python
from res.fm.shell import *
```
and replace some `Shell.method()`calls with calls to the thin wrapper.